src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/Conversions.cs

```diff
@@ -111,9 +130,9 @@ private static MethodSymbol GetDelegateInvokeMethodIfAvailable(TypeSymbol type)
             return methodSymbol;
         }
 
-        public static bool ReportDelegateMethodGroupDiagnostics(Binder binder, BoundMethodGroup expr, TypeSymbol targetType, DiagnosticBag diagnostics)
+        public static bool ReportDelegateOrFuncPtrMethodGroupDiagnostics(Binder binder, BoundMethodGroup expr, TypeSymbol targetType, DiagnosticBag diagnostics)
```

Comment:
Use `FunctionPointer` instead of `FuncPtr` for consistency with rest of the change.